include/Util/Color.hpp

```diff
@@ -49,12 +60,67 @@ class Color : public glm::vec4 {
      */
     SDL_Color ToSdlColor() const {
         return SDL_Color{
-            static_cast<Uint8>(r * 255),
-            static_cast<Uint8>(g * 255),
-            static_cast<Uint8>(b * 255),
-            static_cast<Uint8>(a * 255),
+            static_cast<Uint8>(r),
+            static_cast<Uint8>(g),
+            static_cast<Uint8>(b),
+            static_cast<Uint8>(a),
         };
     }
+
+    /**
+     * @brief Get Color From RGB Values
+     * @param r Red Value (0-255)
+     * @param g Green Value (0-255)
+     * @param b Blue Value (0-255)
+     * @param a Alpha Value (0-255), default is 255
+     * @return Color
+     */
+    static Color FromRGB(Uint8 r, Uint8 g, Uint8 b, Uint8 a = 255);
+
+    /**
+     * @brief Get Color From HSL Values
+     * @param h Hue Value (0-360)
+     * @param s Saturation Value (0-1)
+     * @param l Lightness Value (0-1)
+     * @param a Alpha Value (0-1), default is 1.0
+     * @return Color
+     */
+    static Color FromHSL(float h, float s, float l, float a = 1.0F);
+
+    /**
+     * @brief Get Color From HSV Values
+     * @param h Hue Value (0-360)
+     * @param s Saturation Value (0-1)
+     * @param v Value (0-1)
+     * @param a Alpha Value (0-1), default is 1.0
+     * @return Color
+     */
+    static Color FromHSV(float h, float s, float v, float a = 1.0F);
+
+    /**
+     * @brief Get Color From Hex RGB Value, alpha value is necessary
+     * @param hex Hex Value (0x00000000 - 0xFFFFFFFF)
+     * @return Color
+     */
+    static Color FromHex(Uint32 hex);
+
+    /**
+     * @brief Get Color From Hex RGB String
+     * @param hex Hex String ("00000000" - "FFFFFFFF")
+     * @return Color
+     */
+    static Color FromHex(const std::string &hex);
+
+    /**
+     * @brief Get Color From Name
+     * @param name Color Name, case insensitive, see Util/Color.cpp for
+     * available names
+     * @return Color
+     */
+    static Color FromName(const std::string &name);
```

Comment:
better use enum

Reply:
fixed in bea9e70d23169ad90e2a85878841fa061fdf21ac